### ISSUE TITLE
b.arrange()

### ISSUE THREAD

**Issue by trych** [basiljs/basil.js#109]
I think it would be useful to have the possibility to arrange objects in z-order after creating them.

I am not sure, what the most elegant way would be to implement this. We _could_ use the constants `b.AT_END`, `b.AT_BEGINNING`, `b.AFTER` and `b.BEFORE` to describe `item.sendToBack()`, `item.bringToFront()`, `item.sendBackward()`, `item.sendForward()`.

Or we could use these four constants only if there is a reference object to move the object behind or in front of and then introduce two additional constants `b.BACKWARD`, `b.FORWARD`.

The method could work like this:

`b.arrange(objectToMove, locationOrDirection, [referenceObject]);`

**Comment by fabianmoronzirfas** [comment]
I like the idea and it is needed, but I currently have no idea what would be the best way to implement that.

**Comment by b-g** [comment]
Good call! I would simply go with the wording of indesign EN:
```
b.arrange(objectToMove, locationOrDirection, [referenceObject]);
b.FRONT
b.FORWARD
b.BACKWARD
b.BACK
```
<img width="592" alt="screen shot 2016-10-25 at 06 49 28" src="https://cloud.githubusercontent.com/assets/480224/19683185/39aca5ae-9a7f-11e6-8f8e-d92f600675bc.png">

**Comment by trych** [comment]
> Good call! I would simply go with the wording of indesign EN

Ok, I was not sure, if it is worth it to introduce 4 new constants, but I also think this should be the cleanest implementation. Cool!

**Comment by trych** [comment]
This was implemented in #224.
